v2/tools/asoctl/internal/crd/cleaner.go

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package crd
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"time"
+
+	apiextensionsapi "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1"
```

Comment:
minor: instead of `apiextensionsapi` and `apiextensions`, consider `apiextensions` (for `pkg/apis/apiextensions/v1`) and `apiextensionsclient` (for `pkg/client/clientset/cientset/typed...`)

Reply:
I suggest this mostly because `apiextensionsapi` reads kinda weird to me